content/quiz_meets.md

```diff
@@ -25,5 +32,111 @@
+
+In this bracket the top three teams are involved in a triple elimination, the middle three teams in a double elimination and the last three teams in a single elimination.  This way only those teams that have earned the right through winning will advance to the final quiz.
+
+BRACKET A - This bracket does not require each team to lose to be eliminated.
+Quiz A:    Teams 1, 2, 3
+Quiz B:    Teams 4, 5, 6
+Quiz C:    Teams 7, 8, 9
+Quiz D:    Second Quiz A, Third Quiz A, First Quiz B
+Quiz E:    Second Quiz B, Third Quiz B, First Quiz C
+Quiz F:    Second Quiz D, Third Quiz D, First Quiz E
+
+Quiz G:    Winner Quiz A, Winner Quiz D, Winner Quiz F
+Quiz H:    Same Three Teams in Quiz G
+Quiz I:    If winner of Quiz G does not win Quiz H, Quiz I will have the winners of Quiz G and H (only). The other team will be eliminated
+
+
+
+Tournament Bracket “B”
+Each team in the final nine must lose twice.   A team may make the finals by actually winning only one quiz in this tournament bracket.
+
+BRACKET B
+Quiz A:    Teams 1, 6, 7
+Quiz B:    Teams 2, 5, 8
+Quiz C:    Teams 3, 4, 9
+Quiz D:    First Quiz A, First Quiz C, Second Quiz B
+Quiz E:    First Quiz B, Second Quiz A, Second Quiz C
+Quiz F:    Third Quiz A, Third Quiz B, Third Quiz C
+
+Quiz G:    Third Quiz D, Third Quiz E, First Quiz F
+Quiz H:    Second Quiz D, Second Quiz E, First Quiz G
+
+
+CHAMPIONSHIP QUIZZES
+Quiz I:         First Quiz D, First Quiz E, First Quiz H
+Quiz J:        Second Quiz I, First Quiz I, Third Quiz I
+Quiz K:       Second Quiz J, First Quiz J, Third Quiz J (if needed)
+
+If the same team took third place in Quiz I and J, then Quiz K will be a two-team quiz to determine first and second place.
+
+Quiz L:        Third Quiz K, First Quiz K, Second Quiz K (if needed)
+
+
+Tournament Bracket “C”
+This bracket is a combination of brackets A and B.  A team must win at least two quizzes to obtain a position in the finals.
+BRACKET C
+Quiz A:    Teams 1, 4, 9
+Quiz B:    Teams 2, 5, 7
+Quiz C:    Teams 3, 6, 8
+Quiz D:    First Quiz A, First Quiz B, Second Quiz C
+Quiz E:    Second Quiz A, Second Quiz B, Second Quiz C
+Quiz F:    Third Quiz A, Third Quiz B, Third Quiz C
+
+Quiz G:    Second Quiz D, Third Quiz D, First Quiz E
+Quiz H:    Second Quiz E, Third Quiz E, First Quiz F
+
+
+ Quiz I:    Second Quiz G, Third Quiz G, First Quiz H
+
+CHAMPIONSHIP QUIZZES
+Quiz J:    First Quiz D, First Quiz G, First Quiz I
+Quiz K:        Second Quiz J, First Quiz J, Third Quiz J
+Quiz L:       Second Quiz K, First Quiz K, Third Quiz K (if needed)
+
+If the same team took third place in Quiz J and K, then Quiz L will be a two-team quiz to determine first and second place.
+
+Quiz M:        Third Quiz L, First Quiz L, Second Quiz L (if needed)
 
 ## Championship Quizzes
+A team must win twice to become the champion team.  All three teams will continue to quiz until one team wins twice.
```

Comment:
I consolidated this A LOT.  This top line makes a lot of verbiage unnecessary: e.g. "if no teams wins the first two quizzes, everyone advances to a 3rd quiz"....I mean DUH.